mfaliquot/application/updater.py

```diff
@@ -281,9 +285,17 @@ def postloop_finalize(self, terminated):
 
           _logger.info("Searching for merges...")
           merges = self.seqinfo.find_and_drop_merges()
-          if not merges:
+          if merges:
+               _logger.warning(f"Writing merges to {self.mergefile}")
+               with open(self.mergefile, 'a') as f:
+                    for target, drops in merges:
+                         f.write('_'.join(str(seq) for seq in drops+(target,)) + '\n')
```

Comment:
The `target` needs to be the first in the list. Looking at what was there before it looks like it was also wrong. With your change this should be (or equivalent) to work:
`f.write(target + '_' + '_'.join(str(seq) for seq in drops) + '\n')`

If the lowest leader is the first sequence on each line the common index stuff is working.

P.S.: I deleted two comments I made about this earlier because they were in the wrong places.